packages/sinon/test/unit/lib/SinonSpyCallAssertion.test.ts

```diff
@@ -0,0 +1,266 @@
+/* eslint-disable max-classes-per-file */
+import { AssertionError, expect } from "@assertive-ts/core";
+import { ErrorAssertion } from "@assertive-ts/core/dist/lib/ErrorAssertion";
+import Sinon from "sinon";
+
+import { SinonSpyCallAssertion } from "../../../src/lib/SinonSpyCallAssertion";
+import { sneakyCall } from "../../helpers/common";
+
+class InvariantError extends Error {
+
+  public constructor(message: string) {
+    super(`Invariant: ${message}`);
+    this.name = InvariantError.name;
+  }
+}
+
+class OtherError extends Error {
+
+}
+
+function sayHello(to: string): string {
+  if (to === "") {
+    throw Error("Impossible greet!");
+  }
+
+  if (to === "invariant") {
+    throw new InvariantError("That was unexpected!");
+  }
+
+  return `Hello ${to}`;
+}
+
+describe("[Unit] SinonSpyCallAssertion.test.ts", () => {
```

Comment:
Is this `describe` really needed?
Same for all test files

Reply:
It's not needed, but it's very helpful. Wrapping all the tests within a single `describe(..)` helps to group them as a test suite for the specific file. This encourages to add only tests (and code) related to that file on that suite. Also, all the tests on `assertive-ts/core` are written this way, so it also works in favor of consistency. Another benefit is that if something fails, it's very easy to find the file with the failing test, then copy the file name and find it on the IDE. It's significantly harder to find the file name in the stack trace, check the example below 🙂 

<img width="924" alt="Screenshot 2024-02-09 at 10 58 30" src="https://github.com/stackbuilders/assertive-ts/assets/3087228/4e29ba33-cee4-42c3-8c37-7e45a98a74f9">